src/main/java/build/buildfarm/common/config/ConfigAdjuster.java

```diff
@@ -136,4 +140,42 @@ public static void adjust(BuildFarmServerConfig.Builder builder, ServerOptions o
           "Bytestream timeout not configured.  Setting to: " + defaultDuration.getSeconds() + "s");
     }
   }
+
+  private static void adjustExecuteStageWidth(WorkerConfig.Builder builder) {
```

Comment:
Looks like there is duplication here. It may be better to have a single method that accepts the execute stage width and just set it in the calling method. Something like builder.setExecuteStageWidth(adjustExecuteStageWidth(builder.getExecuteStageWidth());

Reply:
ty, that is way better